refarch-integrations/refarch-cosys-integration/refarch-cosys-integration-example/src/main/resources/application-example.yml

```
@@ -33,3 +21,16 @@ spring:
+refarch:
+  s3:
+    client:
+      document-storage-url: http://localhost:8086
+      enable-security: true
+  cosys:
+    url: https://cosys.example.com/cosys-eai/rest
+    merge:
+      datafile: //multi
+      inputLanguage: Deutsch
+      outputLanguage: Deutsch
+      keepFields: unresolved-ref
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)